search/indexing_api.py

```diff
@@ -33,6 +33,24 @@
 LONG_TYPE = {'type': 'long'}
 
 
+__all__ = [
+    "get_conn",
+    "_index_program_enrolled_users_chunk",
+    "index_program_enrolled_users",
+    "index_users",
+    "remove_program_enrolled_user",
+    "remove_user",
+    "serialize_program_enrolled_user",
+    "refresh_index",
+    "program_enrolled_user_mapping",
+    "create_program_enrolled_user_mapping",
+    "create_mappings",
+    "clear_index",
+    "delete_index",
+    "recreate_index",
+]
```

Comment:
i did a quick google search for iterating through all of a modules functions and saw [this SO answer](http://stackoverflow.com/questions/21885814/how-to-iterate-through-a-modules-functions#21885920). i think that might get us what we want without needing to manually specify `__all__` as we're doing here